cvat/apps/documentation/user_guide.md

```diff
@@ -169,18 +172,39 @@ Go to the [Django administration panel](http://localhost:8080/admin). There you
     The option helps to load high resolution datasets faster.
     Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
 
-    **Select files**. Push this button to select files you want to annotate.
+    **Frame Filter**. Use this option to filter video frames.
+    For example, enter ``step=25`` to leave every twenty fifth frame in the video. Use this option on video files only.
+
+    **Select files or URL list**. If you select `` Remote`` in `` Source``, you'll see a field where you can enter
+    an URL adrresses (one URL per line).
+    Press `` Local`` or `` Share`` in the `` source`` field to choose some files
+    for anotation from your local PC or a network folder respectively.
 
     Push ``Submit`` button and it will be added into the list of annotation tasks.
-    Then, the created task will be displayed on dashboard:
+    Then, the created task will be displayed on a dashboard:
 
-    ![](static/documentation/images/image006.jpg)
+    ![](static/documentation/images/image006_DETRAC.jpg)
 
 1.  The Dashboard contains elements and each of them relates to a separate task. They are sorted in creation order.
     Each element contains: task name, preview, execution status, buttons, and one or more links.
     Each button is responsible for a specific function:
-    - ``Dump Annotation`` — download an annotation file from the task (xml format)
-    - ``Upload Annotation`` — upload an annotation file to the task (xml format)
+    - ``Dump Annotation`` — download an annotation file from a task. Several formats are available:
+      - [CVAT XML 1.1 for video](cvat/apps/documentation/xml_format.md#interpolation)
+      is highlighted if a task has the interpolation mode 
+      - [CVAT XML 1.1 for images](cvat/apps/documentation/xml_format.md#annotation)
+      is highlighted if a task has the annotation mode 
+      - [PASCAL VOC ZIP 1.0](http://host.robots.ox.ac.uk/pascal/VOC/)
+      - [YOLO ZIP 1.0](https://pjreddie.com/darknet/yolo/)
+      - [COCO JSON 1.0](http://cocodataset.org/#format-data)
+      - ``MASK ZIP 1.0`` — archive contains a mask of each frame in the png format and a text file
+      with the value of each color
+      - [TFRecord ZIP 1.0](https://www.tensorflow.org/tutorials/load_data/tf_records)
+    - ``Upload Annotation`` — upload an annotation file to the task. Several formats are available:
+      - [CVAT XML 1.1](cvat/apps/documentation/xml_format.md)
+      - [PASCAL VOC ZIP 1.0](http://host.robots.ox.ac.uk/pascal/VOC/)
```

Comment:
repetitions in those lines

Reply:
@HagegeR 
Why do you think so?

Reply:
line 197 and 205 are the same.
196 and 204
198 and 206
201 and 207
also 192 and 194 are very similar to 203 although there is some added value